docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc

```diff
@@ -1,19 +1,17 @@
 
-Your {page-component-title} repository is defined in your `/etc/yum.repos.d/` directory.
-
+Your {page-component-title} repository is defined in the `/etc/yum.repos.d/` directory.
 ifeval::["{page-component-title}" == "Horizon"]
 The file may be named `opennms-repo-stable-<OSversion>.repo`, but it is not guaranteed to be.
 endif::[]
 ifeval::["{page-component-title}" == "Meridian"]
 The file may be named `{page-component-title}.repo`, but it is not guaranteed to be.
 endif::[]
 
-. Sudo to root.
-
+. Sudo to the root directory.
 ifeval::["{page-component-title}" == "Meridian"]
 . Open your {page-component-title} repository config file with a text editor.
-Change the repository's `baseurl` from the old release family to the new release family.
-The `baseurl` should resemble /https://REPO_USER:REPO_PASS@meridian.opennms.com/packages/2021/stable/rhel8.
+. Change the repository's `baseurl` from the old release family to the new release family.
+The `baseurl` should resemble `/https://REPO_USER:REPO_PASS@meridian.opennms.com/packages/2021/stable/rhel8`.
```

Comment:
Do we have a variable to include here with the latest major release version?
```suggestion
The `baseurl` should resemble `/https://REPO_USER:REPO_PASS@meridian.opennms.com/packages/2022/stable/rhel8`.
```

Reply:
It doesn't seem like we do, but I can make one. :)

Reply:
Done! We now have `latest-meridian-stable` as an attribute.